test/AssemblySet_basic_test.py

```diff
@@ -101,64 +101,66 @@ def test_basic_save_and_get(self):
     assert len(d2["info"]) == INFO_LENGTH
     assert "item_count" in d2["info"][10]
     assert d2["info"][10]["item_count"] == str(N_ASSEMBLIES)
-
-    assert d2["data"]["description"] == "my first assembly set"
+    assert d2["data"]["description"] == set_description
     assert len(d2["data"]["items"]) == N_ASSEMBLIES
 
     item2 = d2["data"]["items"][1]
     assert "info" in item2
     assert len(item2["info"]), INFO_LENGTH
     assert "ref" in item2
-    assert item2["ref"] == self.assembly2ref
+    assert item2["ref"] == assembly_refs[1]
 
     assert "ref_path" in item2
     assert item2["ref_path"] == res["set_ref"] + ";" + item2["ref"]
-    pprint(d2)
 
-def test_save_and_get_of_empty_set(self):
-    setObjName = "nada_set"
+
+def check_empty_set(obj: dict, description: str) -> None:
+    """Check that a set object has zero items in it.
+
+    :param obj: set data as returned by the SetAPI
+    :type obj: dict
+    :param description: description parameter of the set
+    :type description: str
+    """
```

Comment:
For this module, I extracted out the zero items set test into its own function. I'm sure there is other refactoring that could be done throughout this test suite but YOLO!